CodeGeneration/Sources/SyntaxSupport/TypeNodes.swift

```diff
@@ -542,42 +542,17 @@ public let TYPE_NODES: [Node] = [
     elementChoices: [.lifetimeSpecifierArgument]
   ),
 
-  Node(
-    kind: .lifetimeSpecifierArguments,
-    base: .syntax,
-    experimentalFeature: .nonescapableTypes,
-    nameForDiagnostics: nil,
-    documentation: """
-      An optional argument passed to a type parameter.
-
-      ### Example
-      `borrow(data)` in `func foo(data: Array<Item>) -> borrow(data) ComplexReferenceType`
-      """,
-    children: [
-      Child(
-        name: "arguments",
-        kind: .collection(kind: .lifetimeSpecifierArgumentList, collectionElementName: "Arguments"),
-        documentation: """
-          The function parameters that the lifetime of the annotated type depends on.
-          """
-      )
-    ]
-  ),
-
   Node(
     kind: .lifetimeTypeSpecifier,
     base: .syntax,
     experimentalFeature: .nonescapableTypes,
     nameForDiagnostics: "lifetime specifier",
     documentation: "A specifier that specifies function parameter on whose lifetime a type depends",
-    traits: [
-      "Parenthesized"
```

Comment:
I think you need to update some `CodeGeneration` test for this. Could you run the tests in the `CodeGeneration` module?

Reply:
Done.